### DISCUSSION TITLE
Unrestricted union

### DISCUSSION THREAD

**Discussion by stasoid** [litehtml/litehtml#255] [Q&A] [answered]
I am adding more members to property_value and I would like to use union there, but VS2013 doesn't support union members with non-trivial constructors.
Can I upgrade litehtml and litebrowser projects to VS2022?
tig already did this before, but you reverted the change for some reason.
Note: I am not proposing to upgrade the language in which litehtml is written. AFAIK litehtml is written in C++14, and unrestricted union is a C++14 feature ([link](https://en.cppreference.com/w/cpp/language/union)).

**Comment by tordex** [comment, marked as answer]
> Can I upgrade litehtml and litebrowser projects to VS2022?

Yes, let's do it.
